pkg/artifactory/resource/security/resource_artifactory_scoped_token.go

```diff
@@ -100,7 +102,7 @@ func ResourceArtifactoryScopedToken() *schema.Resource {
 							"must be 'applied-permissions/groups:<group-name>[,<group-name>...]'",
 						),
 						validation.StringMatch(
-							regexp.MustCompile(`^artifact:.+:([rwdam\*]|([rwdam]+(,[rwdam]+)))$`),
+							regexp.MustCompile(`^artifact:.+:([rwdam*]|([rwdam]+(,[rwdam]+)))$`),
```

Comment:
Are you sure we don't need `\` to escape the `*`? This is for the literal `*` character in the scope value, not regex.

Reply:
I can add it back in - IJ flagged it as unnecessary

Reply:
Tests pass - let's leave it.